src/user-agent.js

```diff
@@ -1,9 +1,8 @@
 import { arch, platform } from 'os'
 
 import modulePackage from '../package.json'
-import sdkPackage from '../node_modules/sphere-node-sdk/package.json'
 
-const sdkInfo = `node-sdk/${sdkPackage.version}`
+const sdkInfo = `node-sdk/${modulePackage.dependencies['sphere-node-sdk']}`
```

Comment:
Doesn't work because it does [the same lookup](https://github.com/jonschlinkert/load-module-pkg/blob/master/index.js#L18) as I did before.
We could do `'../node_modules/sphere-node-sdk/package.json' || '../../node_modules/sphere-node-sdk/package.json'` to fix it though, thoughts on that?